.gitignore

```diff
@@ -141,7 +141,6 @@ docs/site/
 
 # addon modules
 goldstone/static/addons/
-addon-requirements.txt
```

Comment:
not sure we want to remove this.  if someone checks in the addon-requirements.txt for an addon that is not in the puller's env, they will end up with unnecessary requirements installed.  In the past, the `install_addon/remove_addon` scripts have managed the contents of this file.  It's existence is guaranteed (even if empty) in the docker deployments to ensure that tox has all possible requirements files.